### PR TITLE
jsdoc: add PropTypes namespace for PropType validating methods

### PR DESCRIPTION
**Pull request by machikoyasuda**
## What this PR does
1. Adds `PropType` namespace
2. Adds methods to this namespace

## How that renders

screen shot 2017-10-25 at 3 26 36 pm

### CODE REVIEW

File: lib/api/prop-types.js [hunk 1]
```diff
@@ -2,12 +2,21 @@ import _ from "lodash";
...
+ */
+
 const TagSchema = Schemas.Tag.newContext();
 
 export const PropTypes = {};
 
 /**
- * React Component propType validator for a single Tag
+ * @name Tag
+ * @summary React Component propType validator for a single Tag
```
Comment (mikemurray) [inline lib/api/prop-types.js]: Should `propType` be `PropType` to match the `@file` description above?

Comment (machikoyasuda) [reply lib/api/prop-types.js]: yes! fixed!

Comment (spencern) [reply lib/api/prop-types.js]: In line 17 here it's still showing as `propType` to me

**Comment by machikoyasuda** [comment]
Fixed again and updating master to 1.5.5

**Pull request merged**